config/settings.yml

```diff
@@ -14,10 +14,9 @@ symws:
   login_params: {}
   public_clientID:
 folio:
-  url: <%= ENV['OKAPI_URL'] %>
+  okapi_url: <%= ENV['OKAPI_URL'] %>
   online_service_point_id: 7c5abc9f-f3d7-4856-b8d7-6712462ca007
-folio_graphql:
-  url: <%= ENV['FOLIO_GRAPHQL_URL'] %>
+  graphql_url: <%= ENV['FOLIO_GRAPHQL_URL'] %>
```

Comment:
The GraphQL isn't really part of Folio.  I feel like this is maybe a bit misleading and not as clear as what we had before.

Reply:
If we change it back we should update sul-requests to match mylibrary.

Reply:
Thanks for reviewing. I hear your point. I thought it would be pretty nice if our clients and settings looked as similar as possible between the two apps. I would be willing to change sul-requests to go the other way?